docs/qs-backup-setup.md

```diff
@@ -82,81 +89,155 @@ You should obtain a layout similar to this one (it is fine if you only have one
 
 <img src="/content/images/backup-setup/github-clone-link.png" data-origin="content/images/backup-setup/github-clone-link.png" alt="">
 
-4. In the **command prompt**, type `git clone <git@github.com:......>`. Replace the text between **< >** with the link you copied from GitHub.
-5. Your repository is now synchronized with your computer. Copy your SuperMemo collection in the new folder. Your folder should look similar to the example below (*.gitignore* might be missing):
+4. Copy the link starting with https://
+
+<img src="/content/images/backup-setup/gitsetup-clonehttps.png" data-origin="content/images/backup-setup/gitsetup-clonehttps.png" alt="">
+
+5. In the **command prompt**, type `git clone <https://github.com/......>`. Replace the text between **< >** with the link you copied from GitHub. 'git clone' will create a local copy of your github repository. 
+
+6. A popup will appear. Login with your GitHub account to authenticate.
+
+<img src="/content/images/backup-setup/gitsetup-login.png" data-origin="content/images/backup-setup/gitsetup-login.png" alt="">
+
+7. Your repository is now synchronized with your computer. Copy your SuperMemo collection into the new folder. Your folder should look similar to the example below (*.gitignore* might be missing):
 
 <img src="/content/images/backup-setup/git-local-collection-repository.png" data-origin="content/images/backup-setup/git-local-collection-repository.png" alt="">
 
-6. <a href="/content/data/sm-main-commit.bat" target="_blank" rel="noopener">Download this .bat file</a> and save it in your local repository folder (where your `.git` directory is located). It contains the following commands:
+8. <a href="/content/data/sm-main-commit.bat" target="_blank" rel="noopener">Download this .bat file</a> and save it in your local repository folder (where your `.git` directory is located). It contains the following commands:
 
 ```bat
 git add -A && git commit -m "Update"
 git push
 ```
 
-7. Run `sm-main-commit.bat` (double click). If all went well, your should be able to see your collection in your GitHub repository web page.
+9. Run `sm-main-commit.bat` (double click). If all went well, your should be able to see your collection in your GitHub repository web page.
 
 #### Pushing (*\"saving\"*) your work to GitHub
 
 !> Every time you finish using SuperMemo, make sure to run `sm-main-commit.bat`.
 
 That's all ! Your collection is synchronized online, congratulations !
 
+
+
+#### Optional: Pin commit script to taskbar
+
+To make running sm-main-commit.bat more convenient, you can add it to taskbar. Windows doesn't let you add .bat files to taskbar so instead you can do the following:
+
+1. Right click on sm-main-commit.bat and click [create shortcut ![](/content/images/backup-setup/shortcut-creation.png)](/content/images/backup-setup/shortcut-creation.png '@tooltip-preview').
```

Comment:
FYI: The url here is a very special case, because it starts with a `/`. What this means is that it will look from the root domain (e.g. `https://supermemo.wiki/content/...` if your domain is `supermemo.wiki` and you are using https (as opposed to http)) instead of the relative, current folder (e.g. `https://supermemo.wiki/sma/content/...` if you're in the sma folder).

The reason for this is that we include the backup tutorial both at the root (SM wiki) and in /sma/ (SMA wiki), but we don't want to duplicate the files.